domain/ide/command/command_factory.go

```diff
@@ -63,6 +63,9 @@ func CreateFromCommandData( //nolint:gocyclo // reasonable command dispatch
 	case snyk.GetSettingsSastEnabled:
 		apiClient := snyk_api.NewSnykApiClient(config.CurrentConfig().Engine().GetNetworkAccess().GetHttpClient)
 		return &sastEnabled{command: commandData, apiClient: apiClient}, nil
+	case snyk.GetFeatureFlagStatus:
+		apiClient := snyk_api.NewSnykApiClient(config.CurrentConfig().Engine().GetNetworkAccess().GetHttpClient)
```

Comment:
nitpick: should `config.CurrentConfig().Engine().GetNetworkAccess()` be extracted to a variable?